e2e/pages/CreatePage.js

```diff
@@ -0,0 +1,56 @@
+import { By, until } from 'selenium-webdriver';
+
+module.exports = (url) => (driver) => ({
+    elements: {
+        appLoader: By.css('.app-loader'),
+        titleInput: By.css(`.create-page input[name='title']`),
+        teaserInput: By.css(`.create-page textarea[name='teaser']`),
+        submitButton: By.css(".create-page button[type='submit']"),
+        descInput: By.css(".ql-editor"),
+    },
+
+    navigate() {
+        driver.navigate().to(url);
+            return this.waitUntilDataLoaded();
+    },
+
+    waitUntilVisible() {
+        return driver.wait(until.elementLocated(this.elements.title));
+    },
+
+    waitUntilDataLoaded() {
+        let continued = true;
+        return driver.wait(until.elementLocated(this.elements.appLoader), 400)
+            .catch(() => continued = false) // no loader - we're on the same page !
+            .then(() => continued ? driver.wait(until.stalenessOf(driver.findElement(this.elements.appLoader))) : true)
+            .then(() => driver.sleep(100)); // let some time to redraw
+    },
+
+    setTitleValue(value, clearPreviousValue = true) {
```

Comment:
create a generic `setValue(name, value, clearPreviousValue = true)` instead of repeating the setter code.